owasp-zap/README.md

```diff
@@ -0,0 +1,38 @@
+# OWASP Zed Attack Proxy
+
+[ZAP](https://www.zaproxy.org/) is a security pen testing tool, which this project uses for active scanning for vulnerabilities. To manually run a scan similar to the ones integrated into [our GitHub Actions](../.github/workflows), run `./manual-scan.sh` from a command line. For full details on running the scan locally, see [Manual run ](#manual-run). Below is a brief explanation of the pieces of this integration and what they are for.
+
+# Authenticator
```

Comment:
I can confirm that you are set up in the test environment with permissions like a compact admin for ot/aslp, which currently means read only on license data.

Reply:
@jusdino I think we might need to update this error to be a 403, rather than a 401. The frontend treats 401s as an invalid session and removes session tokens + sends the user back to login.

@jlkravitz I think the frontend is currently showing the full upload UI to compact admins. I'll confirm, but we may need to update the upload UI & main nav for compact admins.

@isabeleliassen Until now, compact admins had been able to get to the license upload UI. It's sounding like we should make an update so that's no longer possible. Does that sound correct?

Reply:
@jsandoval81 I think we just covered this at the backlog planning, but yes, that is correct!

Reply:
@jlkravitz I've created #289 to track this.

Reply:
Wonderful, thanks!